plutus-tx-plugin/test/Strictness/let-strict.pir-readable.golden

```diff
@@ -0,0 +1,17 @@
+let
+  data Unit | Unit_match where
+    Unit : Unit
+  data Bool | Bool_match where
+    True : Bool
+    False : Bool
+in
+\(m : integer) ->
+  let
+    !n : all t. t = /\t -> error {integer -> t} m
```

Comment:
Weird! I have no idea why we're compiling the binding this way.

Reply:
Weird indeed, I'm glad we've found it out, because that bug can really cripple up our users.